src/app/character-tile/CharacterHeaderXP.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import clsx from 'clsx';
+import { t } from 'app/i18next-t';
+
+import { D1Store } from '../inventory/store-types';
+import PressTip from '../dim-ui/PressTip';
+import { percent } from '../shell/filters';
+import { D1ProgressionHashes } from 'app/search/d1-known-values';
+
+function getLevelBar(store: D1Store) {
+  const prestige = store.progression?.progressions.find(
+    (p) => p.progressionHash === D1ProgressionHashes.Prestige
+  );
+  const data = {
+    level: prestige?.level,
+    exp: prestige?.level ? prestige?.nextLevelAt - prestige?.progressToNextLevel : 0,
```

Comment:
I think if `prestige` is `undefined`, you might be better off returning an empty result right away rather than having to use `?.` everywhere.